src/controllers/user.js

```diff
@@ -59,11 +59,18 @@ export const getAll = async (req, res) => {
 }
 
 export const updateById = async (req, res) => {
-  const { cohort_id: cohortId } = req.body
-
-  if (!cohortId) {
-    return sendDataResponse(res, 400, { cohort_id: 'Cohort ID is required' })
+  if (
+    req.body.firstName &&
+    req.body.lastName &&
+    req.body.bio &&
+    req.body.githubUrl
+  ) {
+    req.user.firstName = req.body.firstName
+    req.user.lastName = req.body.lastName
+    req.user.bio = req.body.bio
+    req.user.githubUrl = req.body.githubUrl
+    const updatedUser = await req.user.update()
```

Comment:
You should add a try/catch around the database call - otherwise any excptions from the database could crash the server. See other controller methods in this file for an example,

Reply:
added , let me know if you're happy with it. not very confident with `try...catch`

Reply:
I can't see the changes here - have you pushed?